matrix/common/aws/dynamo_handler.py

```diff
@@ -93,118 +66,95 @@ def _get_dynamo_table_resource_from_enum(self, dynamo_table: DynamoTable):
             return self._state_table
         elif dynamo_table == DynamoTable.OUTPUT_TABLE:
             return self._output_table
-        elif dynamo_table == DynamoTable.CACHE_TABLE:
-            return self._cache_table
 
     def create_state_table_entry(self,
-                                 request_hash: str,
-                                 num_mappers: int,
-                                 format: str=MatrixFormat.ZARR.value):
+                                 request_id: str):
         """
         Put a new item in the DynamoDB table responsible for tracking task execution states and
         counts for a specified request.
 
-        :param request_hash: UUID identifying a filter merge job request.
-        :param num_mappers: Number of mapper lambdas expected to be invoked.
+        :param request_id: UUID identifying a matrix service request.
         :param format: User requested output file format of final expression matrix.
         """
 
         self._state_table.put_item(
             Item={
-                StateTableField.REQUEST_HASH.value: request_hash,
+                StateTableField.REQUEST_ID.value: request_id,
                 StateTableField.EXPECTED_DRIVER_EXECUTIONS.value: 1,
                 StateTableField.COMPLETED_DRIVER_EXECUTIONS.value: 0,
-                StateTableField.EXPECTED_MAPPER_EXECUTIONS.value: num_mappers,
-                StateTableField.COMPLETED_MAPPER_EXECUTIONS.value: 0,
-                StateTableField.EXPECTED_WORKER_EXECUTIONS.value: 0,
-                StateTableField.COMPLETED_WORKER_EXECUTIONS.value: 0,
-                StateTableField.EXPECTED_REDUCER_EXECUTIONS.value: 1,
-                StateTableField.COMPLETED_REDUCER_EXECUTIONS.value: 0,
-                StateTableField.EXPECTED_CONVERTER_EXECUTIONS.value: 0 if format == MatrixFormat.ZARR.value else 1,
+                StateTableField.EXPECTED_QUERY_EXECUTIONS.value: 3,
```

Comment:
Why 3?

Reply:
Lol yes you will. Each matrix service request has 3 queries that need to run currently. The current design has a 1:1 mapping between a query and sqs message. The ecs query runner will pull and run 3 individual queries before scheduling a matrix conversion after the last query finishes execution

Reply:
This allows us to bump up the concurrency of the ecs query runner in terraform without worrying about any unintended consequences on when conversion is scheduled. This also allows us to easily update the driver to add different queries down the road in cases of filtering, etc

Reply:
Cool. What currently are the 3 queries? Is there a design doc I can refer to?

Reply:
There is no design doc currently. The 3 queries are defined in driver.py in this PR

Reply:
I thought you guys already put together the design doc when prototyping redshift (from which the individual units of work were extracted) or are you asking for something different?

Reply:
Just looking to understand what the 3 queries are and what are the potential queries we'd add in the future

Reply:
I'm guessing you mean expression, cell metadata, and gene metadata?

Reply:
Yea. Sorry, I found it later in the PR